src/quo2/components/settings/privacy/style.cljs

```diff
@@ -0,0 +1,96 @@
+(ns quo2.components.settings.privacy.style
+  (:require [quo2.foundations.colors :as colors]))
+
+;;;; Unordered list
```

Comment:
same with this comment, what information do we actually get from this?

Reply:
They're just [heading comments](https://guide.clojure.style/#four-semicolons-for-heading-comments) to help group related parts of the code.
I can remove them, no problem